src/exercises/exerciseTypes/BottomInput.js

```diff
@@ -43,20 +42,15 @@ export default function BottomInput({
     : bookmarksToStudy[0].from_lang;
 
   function handleHint() {
-    if (exerciseType === EXERCISE_TYPES.translateWhatYouHear && firstHint) {
-      setFirstHint(false);
-      onHintUsed();
+    setUsedHint(true);
+    let hint;
+    if (currentInput === targetWord.substring(0, currentInput.length)) {
+      hint = targetWord.substring(0, currentInput.length + 1);
     } else {
-      setUsedHint(true);
-      let hint;
-      if (currentInput === targetWord.substring(0, currentInput.length)) {
-        hint = targetWord.substring(0, currentInput.length + 1);
-      } else {
-        hint = targetWord.substring(0, 1);
-      }
-      setCurrentInput(hint);
-      setMessageToAPI(messageToAPI + "H");
+      hint = targetWord.substring(0, 1);
```

Comment:
What if the user has already written correctly the first 2 letters?

I'd just use the code that is before: 
```

      if (currentInput === targetWord.substring(0, currentInput.length)) {
        hint = targetWord.substring(0, currentInput.length + 1);
      } else {
        hint = targetWord.substring(0, 1);
      }
```

I believe it just adds 1 letter if you already have the first few letters.